origin-dapp/translations/all-messages.json

```diff
@@ -819,6 +819,10 @@
   "ConfirmPublish.afterYouPublishNotice": "After you publish your changes to the blockchain, other users will be able to see that you have verified the following, but they will not be able to see your actual data:",
   "ConfirmPublish.letsDoIt": "Let's do it!",
   "ConfirmPublish.oopsNoWait": "Oops, no wait...",
+  "ConfirmReset.resetIdentity": "Upgrade your Identity?",
```

Comment:
@micahalcorn I came up with those copies. I'd appreciate you putting your PM hat on to help making those better / more explicit for the user :)